safe/impact_function/impact_function.py

```diff
@@ -916,6 +922,24 @@ def run(self):
             message.add(suggestion)
             return ANALYSIS_FAILED_BAD_INPUT, message
 
+        except ProcessingInstallationError:
+            warning_heading = m.Heading(
+                tr('Configuration issue'), **WARNING_STYLE)
+            warning_message = tr(
+                'There is a problem with the Processing plugin.')
+            suggestion_heading = m.Heading(
+                tr('Suggestion'), **SUGGESTION_STYLE)
+            suggestion = tr(
+                'Check in your .qgis2/python/plugins directory that you do '
```

Comment:
I guess on windows it's gonna be `qgis2\python\plugins` but I don't want to take time to customise this message.